src/TestFramework/Config/TestFrameworkConfigLocator.php

```diff
@@ -24,19 +26,33 @@ public function __construct(string $configDir)
     public function locate(string $testFrameworkName, string $customDir = null): string
     {
         $dir = $customDir ?: $this->configDir;
+        $triedFiles = [];
 
-        foreach (['xml', 'yml'] as $extension) {
+        foreach ($this->getDefaultExtensions() as $extension) {
             $conf = sprintf('%s/%s.%s', $dir, $testFrameworkName, $extension);
 
             if (file_exists($conf)) {
                 return realpath($conf);
             }
 
-            if (file_exists($conf . '.dist')) {
-                return realpath($conf . '.dist');
-            }
+            $triedFiles[] = sprintf('%s.%s', $testFrameworkName, $extension);
         }
 
-        throw new \RuntimeException(sprintf('Unable to locate %s.(xml|yml)(.dist) file.', $testFrameworkName));
+        throw LocatorException::multipleFilesDoNoExist(
+            $dir,
+            $triedFiles
+        );
+    }
+
+    private function getDefaultExtensions() : array
+    {
+        return [
+            'xml',
+            'yml',
+            'xml.dist',
+            'yml.dist',
+            'dist.xml',
+            'dist.yml'
```

Comment:
Missing comma here. Just run `composer cs:fix`

Probably the reason why Travis bailed